src/app/portlets/content-types/fields/content-type-field-dragabble-item/content-type-field-dragabble-item.component.scss

```diff
@@ -59,6 +63,9 @@ dot-icon {
     color: gray;
     font-size: $font-size-small;
     line-height: $font-size-small;
+    overflow: hidden;
+    text-overflow: ellipsis;
```

Comment:
Are you sure we need the `overflow: hidden; text-overflow: ellipsis;` in both name and wrapper?

Reply:
you are right `ellipsis` is not needed in the wrapper. `overflow: hidden` is required.